src/IbuMethods.cpp

```diff
@@ -18,7 +18,7 @@
  */
 
 #include "IbuMethods.h"
-#include <cmath>
```

Comment:
Why is this gone? The docs for QtMath says it's just for functions not in cmath.